packages/communication-react/CHANGELOG.beta.md

```diff
@@ -1,9 +1,46 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Tue, 27 Jun 2023 22:07:18 GMT and should not be manually modified.
+This log was last generated on Mon, 24 Jul 2023 17:34:36 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.7.0-beta.2](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.7.0-beta.2)
+
+Mon, 24 Jul 2023 17:34:36 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.7.0-beta.1...@azure/communication-react_v1.7.0-beta.2)
```

Comment:
Could you update 'Compare changes` link for 1.7.0 in the stable changelog?

Reply:
This is beta change log - I guess we should keep comparing to its previous beta version?

Reply:
Yes, but we would need to update it in packages/communication-react/CHANGELOG.stable.md

Reply:
Interesting... For stable I am thinking that we should compare to previous stable release as well (but we are not doing that in our changelog), I will make the change but will raise the concern of this in our chat after this gets merged